docs/getting-started/products/prowler-lighthouse-ai.mdx

```diff
@@ -59,6 +59,12 @@ Prowler Lighthouse AI is powerful, but there are limitations:
 - **NextJS session dependence**: If your Prowler application session expires or logs out, Lighthouse AI will error out. Refresh and log back in to continue.
 - **Response quality**: The response quality depends on the selected LLM provider and model. Choose models with strong tool-calling capabilities for best results. We recommend `gpt-5` model from OpenAI.
 
+## Extending Lighthouse AI
```

Comment:
Leave this section under developer-guide.

We should not remove all the information about how Lighthouse AI is build, we didn't replace it fully by the MCP server. We need the information about how it's build and using the MCP server.